roles/contrail-upload-dmesg/tasks/main.yaml

```diff
@@ -0,0 +1,15 @@
+- name: Gather dmesg output
+  command: dmesg
+  register: dmesg
+  no_log: true
+
+- name: Save dmesg output to a file
```

Comment:
It can probably be replaced with:
```ansible
- name: Write host dmesg output
  delegate_to: localhost
  copy:
    content: "{{ dmesg.stdout }}"
    dest: "{{ zuul.executor.log_root }}/zuul-info"
```

Reply:
Fixed.